ci/deploy_tyk_pro.sh

```diff
@@ -5,34 +5,41 @@ set -e
 NAMESPACE=tykpro-control-plane
 PRODIR=${PWD}/ci/tyk-pro
 ADMINSECRET="54321"
+TIMEOUT=60s
 
-echo "creating namespace ${NAMESPACE}"
-kubectl create namespace ${NAMESPACE}
+
+
+if OUTPUT=$(kubectl get namespaces 2> /dev/null | grep "${NAMESPACE}") ; then
+   echo "namespace ${NAMESPACE} already exists"
+else
+  echo "creating namespace ${NAMESPACE}"
+  kubectl create namespace ${NAMESPACE}
+fi
 
 echo "deploying databases"
 kubectl apply -f "${PRODIR}/mongo/mongo.yaml" -n ${NAMESPACE}
 kubectl apply -f "${PRODIR}/redis" -n ${NAMESPACE}
 
 echo "waiting for redis"
-kubectl wait deployment/redis -n ${NAMESPACE} --for condition=available --timeout=60s
+kubectl wait deployment/redis -n ${NAMESPACE} --for condition=available --timeout=${TIMEOUT}
 echo "waiting for mongo"
-kubectl wait deployment/mongo -n ${NAMESPACE} --for condition=available --timeout=60s
+kubectl wait deployment/mongo -n ${NAMESPACE} --for condition=available --timeout=${TIMEOUT}
```

Comment:
Depending on circumstances, it might be better to use `--all` rather than specifying mongo and redis separately. This might work while waiting for gw and dash below as well.

Reply:
solved